debian/control

```diff
@@ -79,6 +79,7 @@ Vcs-Browser: https://salsa.debian.org/gnome-team/gnome-shell
 Package: gnome-shell
 Architecture: linux-any
 Depends: adwaita-icon-theme,
+         dconf-cli,
```

Comment:
Commit message claims "Used in the postinst script" but I can't see any use of it.